apps/src/aichat/views/chatMessage.module.scss

```diff
@@ -0,0 +1,39 @@
+@import 'color.scss';
+
+.name {
+    font-family: "Gotham 5r", sans-serif;
```

Comment:
Probably a question for @levadadenys - should we use a `<Typography>` component in place of this?

Reply:
I'll defer to @levadadenys . Thanks!

Reply:
Added `Typography` and will ask @levadadenys to take a look when he gets back so he can tweak as he sees fit.